tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -17,6 +17,48 @@ jobs:
       displayName: CPU EP, Build and Test on Android Emulator
     - script: /bin/bash tools/ci_build/github/android/run_nnapi_code_coverage.sh $(pwd)
       displayName: NNAPI EP, Build, Test and Get Code Coverage on Android Emulator
+    - task: PublishPipelineArtifact@0
+      displayName: 'Publish code coverage report'
+      inputs:
+        artifactName: "coverage_rpt.txt"
+        targetPath: '$(Build.SourcesDirectory)/build_nnapi/Debug/coverage_rpt.txt'
+        publishLocation: 'pipeline'
     - script: /bin/bash tools/ci_build/github/linux/ort_minimal/nnapi_minimal_build_minimal_ort_and_run_tests.sh $(pwd)
       # Build Minimal ORT with NNAPI and reduced Ops, run unit tests on Android Emulator
       displayName: Build Minimal ORT with NNAPI and run tests
+
+- job: Update_Dashboard
+  workspace:
+    clean: all
+  pool: 'Linux-CPU'
+  dependsOn: Android_CI
+  condition: succeeded()
+  steps:
+
+    - template: templates/set-version-number-variables-step.yml
+
+    - task: DownloadPipelineArtifact@0
+      displayName: 'Download code coverage report'
+      inputs:
+        artifactName: 'coverage_rpt.txt'
+        targetPath: '$(Build.BinariesDirectory)'
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '3.x'
+    - task: CmdLine@2
+      inputs:
+        script: |
+          set -e
+          sudo apt-get install -y python3-pip
```

Comment:
is sudo and installing pip required? i think it may not be on a hosted agent after UsePythonVersion

Reply:
I ran into a python error unknown module pip when I first ran it. I had to add the command to install pip.